source/Octopus.Tentacle.Tests.Integration/ClientFileTransfersAreNotRetried.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Halibut;
+using NUnit.Framework;
+using Octopus.Tentacle.Tests.Integration.Support;
+using Octopus.Tentacle.Tests.Integration.Util.Builders;
+using Octopus.Tentacle.Tests.Integration.Util.Builders.Decorators;
+using Octopus.Tentacle.Tests.Integration.Util.TcpTentacleHelpers;
+
+namespace Octopus.Tentacle.Tests.Integration
+{
+    public class ClientFileTransfersAreNotRetried : IntegrationTest
```

Comment:
It would be good to add why e.g. WhenRetriesAreDisabled

Reply:
And the negative to the existing tests `source/Octopus.Tentacle.Tests.Integration/ClientFileTransfersAreRetried.cs` to make it clear why there is/isn't retry behavour